src-docs/src/views/auto_refresh/auto_refresh_button.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+
+import { EuiAutoRefreshButton, OnRefreshChangeProps } from '../../../../src';
+
+export default () => {
+  const [refreshInterval, setRefreshInterval] = useState(1000);
+  const [isPaused, setIsPaused] = useState(true);
```

Comment:
As someone not really familiar with the concept of auto refresh I was pretty confused by these demos/playgrounds initially. I know the default is to `isPaused: true` which sets the components to essentially 'off', but could we consider having the documentation set to an initial state of `isPaused: false` for the purposes of discovery/playing around with the main component functionality?

For example, I tested the `shortHand` prop initially by clicking the playground toggle, and was super confused when nothing happened / the time unit dropdown did not change from `seconds` to `s`. It was only after turning `isPaused` to false that I realized the text changing to shorthand was on the button, not in the dropdown.

The below suggestion is just one example, I think the playground would need extra customization as well

```suggestion
  const [isPaused, setIsPaused] = useState(false);
```

Reply:
I've updated this button example specifically to be both `!isPaused` and `shortHand` by default, but I didn't change the playground because I like to keep these at their default as much as possible unless they'll error without a certain prop, usually handlers.